static/app/components/core/segmentedControl/index.tsx

```diff
@@ -236,7 +242,7 @@ function Segment<Value extends string>({
 }
 
 const GroupWrap = withChonk(
-  styled('div')<{priority: Priority; size: FormSize}>`
+  styled('div')<{listSize: number; priority: Priority; size: FormSize}>`
```

Comment:
Is `listSize` used anywhere? Maybe intended as a negative margin-right that counteracts the `transform`?

Reply:
It's used in the chonk version and types must align

Reply:
Ah gotcha, thanks for clarifying!